### PR TITLE
Javascript-javascript2-week3

### PR DESCRIPTION
**Pull request by santhoshboinapally**
Creating pull request for js2-week3

### CODE REVIEW

File: javascript/javascript2/week3/fastestpresser.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    left: 6%;
+    margin-left: -10px;
+    top: 0;
+}
+
+.game {
+    font-size: 3em;
+    color: rgb(255, 0, 149);
+    font-style: italic;
+}
```
Comment (MirSzabo) [inline javascript/javascript2/week3/fastestpresser.css:56]: I love the design you created 💯 
anyway it is little bit confusing you put L to the left but the key is located on the right part of the keyboard and opposite with S
or it is by purpose? :)

File: javascript/javascript2/week3/fastestpresser.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+// counting for L
+let totalL = 0;
+const keyL = document.querySelector("#countL");
+const pressL = (event) => {
+    if (event.keyCode == 76) {
+        totalL++;
+        keyL.innerHTML = totalL;
+    }
+};
+
```
Comment (MirSzabo) [inline javascript/javascript2/week3/fastestpresser.js:10]: you should count how many times button is pressed but if I hold down button it is counting many presses

File: javascript/javascript2/week3/hw.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        console.log("First Array Function");
+    },
+    function() {
+        console.log("Second Arrya Function");
+    },
+    function() {
+        console.log("Third Array Function");
+    },
+];
+for (let i = 0; i < arrFunction.length; i++) console.log(arrFunction[i]);
```
Comment (MirSzabo) [inline javascript/javascript2/week3/hw.js:116]: you have to call the function 
```
for (let i = 0; i < arrFunction.length; i++) {
  arrFunction[i]();
}
```

File: javascript/javascript2/week3/fastestpresser.js (hunk 2 above)
Comment (santhoshboinapally) [reply javascript/javascript2/week3/fastestpresser.js:10]: updated "keydown" event with "keyup" event .Thankyou

File: javascript/javascript2/week3/fastestpresser.css (hunk 1 above)
Comment (santhoshboinapally) [reply javascript/javascript2/week3/fastestpresser.css:56]: Thankyou, This was a mistake. I updated the code.

**Pull request merged**